lecture_4/02_bets_api/internal/api/controllers/validators/betDtoIdValidator.go

```diff
@@ -0,0 +1,20 @@
+package validators
+
+// BetDtoIdValidator validates event update requests.
+type BetDtoIdValidator struct{}
+
+// NewBetDtoIdValidator creates a new instance of BetDtoIdValidator.
+func NewBetDtoIdValidator() *BetDtoIdValidator {
+	return &BetDtoIdValidator{}
+}
+
+// IdIsValid checks if event update is valid.
+// Id is not empty
+// Outcome is `lost`or `won`
```

Comment:
Comment doesn't really match the implementation (e.g. why is "outcome" mentioned here?). Also "Id is not empty" sounds like an unfinished sentence.